### PR TITLE
Documentrcudbinfo

### PR DESCRIPTION
**Pull request by jshum2479**
Add usage documentation for RCUDbInfo section in the the domainInfo of the model

### CODE REVIEW

**Comment by ddsharpe** [comment]
Needs documentation review

File: site/rcuinfo.md [hunk 1]
```diff
@@ -1,6 +1,6 @@
 ## Specifying RCU connection information in the model
 
-During creation or updating a domain. There is new section RCUDbInfo in the model under domainInfo section for specifying RCU database connection information without using the command line options -rcu_database and -rcu_prefix.  This is created to support database where the connection string is more complicated and require extra options.
+During creation or updating a domain. There is new section RCUDbInfo in the model under domainInfo section for specifying RCU database connection information without using the command line options -rcu_database and -rcu_prefix.  This is created to support database where the connection string is more complicated and requires extra options.
```
Comment (rosemarymarano) [inline site/rcuinfo.md]: Use tic marks to apply `code` font to words and commands, such as `RCUDbInfo`, `domainInfo`, `-rcu_database`, 
 `-rcu_prefix`
**Suggested re-write**:
During creating or updating a domain, there is new section `RCUDbInfo` in the model, under the `domainInfo` section, for specifying RCU database connection information without using the command-line options 
`-rcu_database` and `-rcu_prefix`.  Use this to support a database where the connection string is more complicated and requires extra options.

File: site/create.md [hunk 2]
```diff
@@ -19,6 +19,9 @@ The Create Domain Tool understands three domain types: `WLS`, `RestrictedJRF`, a
 
 To have the Create Domain Tool run RCU, simply add the `-run_rcu` argument to the previous command line and the RCU schemas will be automatically created.  Be aware that when the tool runs RCU, it will automatically drop any conflicting schemas that already exist with the same RCU prefix prior to creating the new schemas!
 
+It is also possible to specify the connection information in the model instead of 
+using the command line.  This is especially easier for complicated jdbc connection string such as RAC or Oracle Autonomous Transaction Processing Cloud Service databases.  For information of how to use it, refer to [Specifying RCU connection information in the model](rcuinfo.md)
```
Comment (rosemarymarano) [inline site/create.md]: complicated jdbc connection string such as -> complicated JDBC connection strings, such as

Comment (rosemarymarano) [reply site/create.md]: For information of how to use it -> For information on how to use it

File: site/rcuinfo.md [hunk 3]
```diff
@@ -0,0 +1,46 @@
...
+        rcu_admin_user : admin
+        tns.alias : dbatp_tp
+        javax.net.ssl.keyStorePassword : xxxx
+        javax.net.ssl.trustStorePassword : xxxx
+```           
+The database wallet can be included in the archive file under atpwallet/ structure
+
+atpwallet/Walletxyz.zip
+        
+Using createDomain tool with the -run_rcu flag will create the rcu schemas against the ATP database and configure the JRF domain
```
Comment (rosemarymarano) [inline site/rcuinfo.md]: Using createDomain tool with the -run_rcu flag will create the rcu schemas -> Using the Create Domain Tool with the `-run_rcu` flag, will create the RCU schemas

File: site/rcuinfo.md [hunk 4]
```diff
@@ -0,0 +1,46 @@
...
+with the ATP database.  For example:
+
+    weblogic-deploy/bin/updateDomain.sh -oracle_home /u01/wls12213 -domain_type JRF -domain_home /u01/data/domains/demodomain -archive_file DemoDomain.zip -run_rcu 
+
+For non-ATP database, use the following example:
```
Comment (rosemarymarano) [inline site/rcuinfo.md]: For non-ATP database, -> For a non-ATP database,

File: site/rcuinfo.md [hunk 5]
```diff
@@ -0,0 +1,46 @@
...
+```yaml
+domainInfo:
+    RCUDbInfo:
+        useATP : 0
+        rcu_prefix : DEV
+        rcu_schema_password : xxxxx
+        rcu_admin_password : xxxx
+        rcu_db_conn_string : 'dbhost:1521/pdborcl'
+```        
+Specifying RCU variables to the repository creation utility can now be included in the RCUDbInfo section with the key:
```
Comment (rosemarymarano) [inline site/rcuinfo.md]: RCUDbInfo -> `RCUDbInfo`

File: site/rcuinfo.md [hunk 6]
```diff
@@ -0,0 +1,46 @@
+## Specifying RCU connection information in the model
+
+During creation or updating a domain. There is new section RCUDbInfo in the model under domainInfo section for specifying RCU database connection information without using the command line options -rcu_database and -rcu_prefix.  This is created to support database where the connection string is more complicated and requires extra options.
+
+For example, in order to use Oracle Autonomous Transaction Processing Cloud Database for the JRF domain, specify the following information in the model:
```
Comment (rosemarymarano) [inline site/rcuinfo.md]: in order to use Oracle Autonomous Transaction Processing Cloud Database -> in order to use the Oracle Autonomous Transaction Processing Cloud Database

File: site/rcuinfo.md [hunk 7]
```diff
@@ -0,0 +1,46 @@
...
+    RCUDbInfo:
+        useATP : 1
+        rcu_prefix : DEV
+        rcu_schema_password : xxxxx
+        rcu_admin_user : admin
+        tns.alias : dbatp_tp
+        javax.net.ssl.keyStorePassword : xxxx
+        javax.net.ssl.trustStorePassword : xxxx
+```           
+The database wallet can be included in the archive file under atpwallet/ structure
```
Comment (rosemarymarano) [inline site/rcuinfo.md]: under atpwallet/ structure -> under the `atpwallet` (?)directory

**Pull request closed without merging**